graphql/http_handler.go

```diff
@@ -57,12 +58,14 @@ func Handler(apiURL string) func(w http.ResponseWriter, r *http.Request) {
 			queryPath = fieldCtx.Path().String()
 			args = fieldCtx.Args
 		}
-		grip.Error(message.WrapError(err, message.Fields{
-			"path":    "/graphql/query",
-			"query":   queryPath,
-			"args":    args,
-			"request": gimlet.GetRequestID(ctx),
-		}))
+		if !strings.HasSuffix(err.Error(), ": context canceled") {
```

Comment:
err.Error() will panic if err is nil so we need to check that explicitly (message.WrapError noops if err is nil which is why we didn't need to do taht before)

Also preference to make this a string a const somewhere (and just checking for "context canceled" would be cleaner but idk if there's something you're looking to avoid)

Reply:
You can use errors.is instead of the string comparison.

Reply:
I think you should be able to use `errors.is` instead of doing a string comparison. 

https://pkg.go.dev/errors